src/Kiota.Builder/CodeRenderers/CodeRenderer.cs

```diff
@@ -44,7 +47,11 @@ public async Task RenderCodeNamespaceToFilePerClassAsync(LanguageWriter writer,
                 case CodeFunction:
                 case CodeInterface:
```

Comment:
just add the codeFile case here since the body of the case is the same. This will avoid duplication.